### DISCUSSION TITLE
.net core的一些疑问

### DISCUSSION THREAD

**Discussion by JaychengLeo** [BYJRK/DotNet-Discussions#213] [WPF 开发]
一直以来我都是用的.net framework框架，看了大佬几期视频后，我尝试将项目迁移至.net core。因为我主要做上位机，用.net framework可以用costura.fody等将所有dll文件打包，这样就可以得到一个exe文件，比较方便用户使用。我尝试使用.net core，不知道是不是我没有正确操作或者怎么的，目前没能做到这一点。还望大佬赐教，万分感谢！

**Comment by BYJRK** [comment]
Costura.Fody 的 [README](https://github.com/Fody/Costura?tab=readme-ov-file#-read-this--package-is-in-maintenance-mode--read-this-) 里面就写了两种 Core 可以使用的方式

**Reply by JaychengLeo** [reply]
首先，非常感谢大佬的回复。
我去看了下，说的好像是推荐用微软自带的工具集。然后跳转一看就是我之前尝试的发布成单文件的方式。这种方式始终会存在如下图所示的几个文件
![image](https://github.com/BYJRK/DotNet-Discussions/assets/67141458/375a71f9-2662-4f24-ae6b-e287ba7ac019)
之前没深入研究，今天找了下资料发现可以借助warp-packer.exe实现，不过美中不足就是运行后会有个额外的gcf.exe窗口。
